.github/workflows/ci.yaml

```diff
@@ -34,8 +34,8 @@ jobs:
       with:
         node-version: '18.19.0'
     - run: yarn install
-    - run: yarn oss
-    - run: git diff --exit-code && git diff --cached --exit-code || (echo "Please run 'yarn oss' and commit the result." && exit 1)
+    - run: yarn cli integrity
+    - run: git diff --exit-code && git diff --cached --exit-code || (echo "Please run 'yarn cli integrity' and commit the result." && exit 1)
```

Comment:
Shouldn't we run both `oss` and `integrity`?

Reply:
`oss` is ran at the end of `integrity` part of the `runAutoFixes`.